src/test/java/easytests/core/mappers/QuizzesMapperTest.java

```diff
@@ -1,148 +1,106 @@
+
     @Autowired
     private QuizzesMapper quizzesMapper;
 
     @Test
-    public void testFind() throws Exception {
-
-        final QuizEntity quiz = this.quizzesMapper.find(1);
+    public void testFindAll() throws Exception {
 
-        Assert.assertEquals((long) 1, (long) quiz.getId());
-        Assert.assertEquals("test_invite_code1", quiz.getInviteCode());
-        Assert.assertEquals(LocalDateTime.of(2003,2,1,0,0,0), quiz.getStartedAt());
-        Assert.assertEquals(LocalDateTime.of(2003,3,1,0,0,0), quiz.getFinishedAt());
-        Assert.assertEquals(false, quiz.getCodeExpired());
-    }
+        final List<QuizEntity> quizFoundedEntities = this.quizzesMapper.findAll();
 
-    @Test
-    public void testFindAll() throws Exception {
+        Assert.assertNotNull(quizFoundedEntities);
+        Assert.assertEquals((long) 3, (long) quizFoundedEntities.size());
 
-        final List<QuizEntity> quizEntities = this.quizzesMapper.findAll();
+        Integer index = 0;
+        for(QuizEntity quizEntity: quizFoundedEntities){
+            final QuizEntity quizFixtureEntity= this.quizzesSupport.getEntityFixtureMock(index);
 
-        Assert.assertNotNull(quizEntities);
-        Assert.assertEquals((long) 3, (long) quizEntities.size());
+            this.quizzesSupport.assertEquals(quizFixtureEntity, quizEntity);
+            index++;
+        }
 
     }
 
     @Test
-    public void testIssueNotNull() throws Exception {
-
-        final List<QuizEntity> quizEntities = this.quizzesMapper.findByIssueId(4);
+    public void testFind() throws Exception {
 
-        Assert.assertNotNull(quizEntities);
-        Assert.assertEquals(0, quizEntities.size());
+        final QuizEntity quizFixtureEntity = this.quizzesSupport.getEntityFixtureMock(1);
+        final QuizEntity quizFoundedEntity = this.quizzesMapper.find(quizFixtureEntity.getId());
 
+        this.quizzesSupport.assertEquals(quizFixtureEntity, quizFoundedEntity);
     }
 
     @Test
     public void testFindByIssueId() throws Exception {
+        final List<QuizEntity> quizzesFixtureEntities = new ArrayList<>();
+        quizzesFixtureEntities.add(this.quizzesSupport.getEntityFixtureMock(1));
 
-        final List<QuizEntity> quizEntities = this.quizzesMapper.findByIssueId(3);
+        final List<QuizEntity> quizzesFoundedEntities = this.quizzesMapper.findByIssueId(2);
 
-        Assert.assertEquals(1, quizEntities.size());
-        Assert.assertEquals("test_invite_code3", quizEntities.get(0).getInviteCode());
-        Assert.assertEquals(LocalDateTime.of(2003,2,1,0,0,0), quizEntities.get(0).getStartedAt());
-        Assert.assertEquals(LocalDateTime.of(2003,3,1,0,0,0), quizEntities.get(0).getFinishedAt());
-        Assert.assertEquals(true, quizEntities.get(0).getCodeExpired());
+        Assert.assertEquals(1, quizzesFoundedEntities.size());
+        this.quizzesSupport.assertEquals(quizzesFixtureEntities.get(0), quizzesFoundedEntities.get(0));
```

Comment:
compare as list pls(i.e. in for loop), if there is issue id that produce two elements use it.